vividus-plugin-kafka/src/test/java/org/vividus/steps/kafka/KafkaStepsIntegrationTests.java

```diff
@@ -198,8 +203,20 @@ void shouldProduceEventWithKey() throws InterruptedException, ExecutionException
         kafkaSteps.stopKafkaListener(CONSUMER);
         kafkaSteps.processKafkaEvents(QueueOperation.PEEK, CONSUMER, SCOPES, VARIABLE_NAME);
 
-        assertThat(logger.getLoggingEvents(), is(List.of(info(LISTENER_STARTED), info(LISTENER_STOPPED),
-                info("Saving events with the keys: {}", "key1, <no key>, <no key>, key2, key3, <no key>"))));
+        List<LoggingEvent> events = logger.getLoggingEvents();
+        assertThat(events, hasSize(3));
+        assertEquals(info(LISTENER_STARTED), events.get(0));
+        assertEquals(info(LISTENER_STOPPED), events.get(1));
+        LoggingEvent keysLogEvent = events.get(2);
+        assertEquals(Level.INFO, keysLogEvent.getLevel());
+        assertEquals("Saving events with the keys: {}", keysLogEvent.getMessage());
+        List<Object> arguments = keysLogEvent.getArguments();
+        assertThat(arguments, hasSize(1));
+        List<String> keys = Stream.of(arguments.get(0).toString().split(",")).map(String::strip).sorted()
```

Comment:
any reasons to sort ?

Reply:
different order of events

Reply:
I mean there are matchers for like hasItesm containsInAnyOrder etc

Reply:
these matchers are good if we have distinct values, in out case we have duplicates